src/Widget/Controller/WidgetController.php

```diff
@@ -301,6 +301,7 @@ public function renderDetailPage(Request $request, WidgetPageInterface $widgetPa
         $data = [
             'data' => $this->renderer->renderDetailPage($widget, $preferredLanguage),
         ];
+        $data['data'] = mb_convert_encoding($data['data'], "UTF-8", "UTF-8");
```

Comment:
Can you add a comment to clarify why this is necessary?
It seems you're converting from UTF-8 to UTF-8, which wouldn't change anything in theory?
If there is an interesting side effect, it would be best to document it so no one thinks to remove this code later on.

Reply:
@bertramakers there was an issue with a character which made the application crash see ticket. I did some research on stackoverflow and this seems to fix the problem.

Reply:
Sure, but can you document that with an inline comment? It's not clear by just looking at the code what the point of this line is. Other than that it looks good to me.